concepticondata/conceptlists/Clark-2008-757.tsv

```diff
@@ -0,0 +1,789 @@
+Clark-2008-757-718	718	to shoot	1172	SHOOT
+Clark-2008-757-719	719	to hunt	1435	HUNT
+Clark-2008-757-720	720	to fight	1423	FIGHT
+Clark-2008-757-721	721	to exchange	2509	EXCHANGE
+Clark-2008-757-722	722	to disappear	485	DISAPPEAR
+Clark-2008-757-723	723	to flow	2003	FLOW
+Clark-2008-757-724	724	to float	1574	FLOAT
+Clark-2008-757-725	725	to sink	1088	SINK (DESCEND)
+Clark-2008-757-726	726	to twinkle	2889	TWINKLE
+Clark-2008-757-727	727	to cook (rice)	1100	COOK (SOMETHING)
+Clark-2008-757-728	728	to wash rice	1453	WASH
+Clark-2008-757-729	729	to cut into pieces	2149	CHOP (INTO PIECES)
+Clark-2008-757-730	730	to cut into slices		
+Clark-2008-757-731	731	to chop/mince	2149	CHOP (INTO PIECES)
+Clark-2008-757-732	732	to knead (a dough)	274	KNEAD
+Clark-2008-757-733	733	to pound (garlic)	354	POUND
+Clark-2008-757-734	734	to put in (ingredients)	998	PUT
+Clark-2008-757-735	735	to pour over	1487	POUR
+Clark-2008-757-736	736	to squeeze	1414	SQUEEZE
+Clark-2008-757-737	737	to crack an egg		
+Clark-2008-757-738	738	to boil	792	BOIL (OF LIQUID)
+Clark-2008-757-739	739	to stir fry, to fry	2162	STIR FRY
+Clark-2008-757-740	740	to fry in fat/oil	991	FRY
+Clark-2008-757-741	741	to wash	1453	WASH
+Clark-2008-757-742	742	to write	1672	WRITE
+Clark-2008-757-743	743	to tear	1735	TEAR (SHRED)
+Clark-2008-757-744	744	to talk	2708	TALK
+Clark-2008-757-745	745	to stoke		
+Clark-2008-757-746	746	to soak	2069	SOAK (VERB)
+Clark-2008-757-747	747	to smell	2124	SMELL
+Clark-2008-757-748	748	to slash		
+Clark-2008-757-749	749	to set trap		
+Clark-2008-757-750	750	to scrape	1767	SCRAPE
+Clark-2008-757-751	751	to scold	299	SCOLD
+Clark-2008-757-752	752	to run	1519	RUN
+Clark-2008-757-753	753	to roar	2876	ROAR (OF SEA)
+Clark-2008-757-754	754	to ride horse	367	RIDE
+Clark-2008-757-755	755	to rest	168	REST
+Clark-2008-757-756	756	to remember	707	REMEMBER
+Clark-2008-757-757	757	to pull gr	1455	PULL
+Clark-2008-757-758	758	to pat	2133	STRIKE OR BEAT
+Clark-2008-757-759	759	to open mr		
+Clark-2008-757-760	760	to mow	190	MOW
+Clark-2008-757-761	761	to marry w		
+Clark-2008-757-762	762	to marry d		
+Clark-2008-757-763	763	to like	1117	LIKE
+Clark-2008-757-764	764	to kill	1417	KILL
+Clark-2008-757-765	765	to hone kn		
+Clark-2008-757-766	766	to hide	2486	HIDE
+Clark-2008-757-767	767	to go	695	GO
+Clark-2008-757-768	768	to give birth	1195	GIVE BIRTH
+Clark-2008-757-769	769	to fry	991	FRY
+Clark-2008-757-770	770	to fear	1419	FEAR (BE AFRAID)
+Clark-2008-757-771	771	to fall down	1280	FALL
+Clark-2008-757-772	772	to explode		
+Clark-2008-757-773	773	to sun dry	3364	DRY IN SUN
+Clark-2008-757-774	774	to draw water	1911	DRAW (WATER)
+Clark-2008-757-775	775	to drain		
+Clark-2008-757-776	776	to cut w/ scissors	2155	CUT (WITH SCISSORS)
+Clark-2008-757-777	777	to cut w/ knife	2156	CUT (WITH KNIFE)
+Clark-2008-757-778	778	to crow	2074	CROW (VERB)
+Clark-2008-757-779	779	to cook	1100	COOK (SOMETHING)
+Clark-2008-757-780	780	to chew	321	CHEW
+Clark-2008-757-781	781	to chase	1047	PURSUE
```

Comment:
This verb is inconsistently mapped to PURSUE or HUNT. A typical issue for our clean-up bucket list ;-)